### DISCUSSION TITLE
Digest email

### DISCUSSION THREAD

**Discussion by Sealjay** [Green-Software-Foundation/discussions#4] [Ask Anything]
Love the idea of a new GitHub Digest email!

- I wanted to share these integrations too, as they really help me keep track of things in GitHub:

- GitHub for teams - https://teams.github.com/ - I subscribe to issues and discussions

<img width="987" alt="image" src="https://user-images.githubusercontent.com/19361656/217321889-3116eac7-d458-4c68-84c3-5d1fb9abce1b.png">

- GitHub for slack - https://github.com/integrations/slack#subscribe-to-an-organization-or-a-repository

<img width="817" alt="image" src="https://user-images.githubusercontent.com/19361656/217321956-5a6bb2fa-0d82-4f21-97fe-fcab5b516521.png">

**Comment by jawache** [comment]
These are great ideas, will make a note to add them to our getting started guide.
